modules/meeting/config/routes.rb

```diff
@@ -27,11 +27,11 @@
 #++
 
 OpenProject::Application.routes.draw do
-  scope 'projects/:project_id' do
-    resources :meetings, only: %i[new create index]
+  resources :projects, only: %i[] do
```

Comment:
This might be a good candidate to use [shallow route nesting](https://guides.rubyonrails.org/routing.html#shallow-nesting).